src/epics/devices.js

```diff
@@ -47,26 +54,35 @@ const initialDevices = [
 
 // const initialDevices = [];
 
+export const announceSyncEpic = (action$: ActionsObservable<any>) =>
+  action$
+    .ofType(ANNOUNCE_SYNC)
+    .filter(action => action.status === 'Start')
+    .flatMap(() => Sync.announce());
```

Comment:
hmmm so this will break cause you're not passing back an action. You have a success reducer for announceSynce so you might want to just do `Sync.announce().map((resp) => syncAnnounce(action.meta, resp))`